edkrepo/common/humble.py

```diff
@@ -57,7 +57,12 @@
 FETCHING = 'Fetching latest code for {0} from {1} branch ...'
 NO_SYNC_DETACHED_HEAD = 'No need to sync repo {0} since it is in detached HEAD state'
 SYNC_MANIFEST_UPDATE = 'To update to the latest manifest please run edkrepo sync --update-local-manifest. {}'.format(Fore.RESET)
-SYNC_REMOVE_LIST_END_FORMATTING = '{}'.format(Fore.RESET)
+SYNC_REMOVE_LIST_END_FORMATTING = '{}'.format(Style.RESET_ALL)
+SYNC_MOVE_FAILED = '''{}{}WARNING:{}{} Moving {{}} to {{}} failed.

```

Comment:
We should make this a little more readable. Adding labels like {{old_dir}} and {{new_dir}} should make this much more readable.